src/views/Settings.vue

```diff
@@ -123,7 +124,39 @@ export default {
       mdiWeb,
       mdiAccountReactivate
     },
-    tab: "first"
+    tab: "first",
+    tabs: [
+      {
+        icon: mdiWeb,
+        text: "app.language",
+        value: "first",
+        component: LanguageSettings
+      },
+      {
+        icon: mdiFormatSection,
+        text: "app.gdpr",
+        value: "second",
+        component: GDPR
+      },
+      {
+        icon: mdiBadgeAccountHorizontal,
+        text: "personnelNumber.label",
+        value: "third",
+        component: PersonnelNumberForm
+      },
+      {
+        icon: mdiAccountRemove,
+        text: "app.account",
+        value: "fourth",
+        component: DeleteAccount
+      },
+      {
+        icon: mdiAccountReactivate,
+        text: "app.checkoutUser",
```

Comment:
Formating of the last Card, although only visible for admins, is a bit off

![image](https://github.com/user-attachments/assets/840695df-6290-4140-ad16-973e4461ab70)

For Completeness please fix this.